### PR TITLE
Help center articles about learner records

### PR DESCRIPTION
**Pull request by grantgoodmanedX**
## [DOC-3949](https://openedx.atlassian.net/browse/DOC-3949)

Initial help center articles about sharing learner records 

### Reviewers

Possible roles follow. The PR submitter checks the boxes after each reviewer finishes and gives :+1:. 

- [ ] Subject matter expert: @mikix 
- [ ] Subject matter expert: 
- [ ] Doc team review (sanity check, copy edit, or dev edit?): @edx/doc
- [ ] Product review: @marcotuts 

### Testing

- [ ] Ran ./run_tests.sh without warnings or errors

### HTML Version (optional)

- [ ] Build an RTD draft for your branch and add a link here

### Post-review

- [ ] Squash commits

### CODE REVIEW

**Comment by mikix** [comment]
The UI mockups (https://projects.invisionapp.com/share/ADK3TZQX3GP) show a single general entry point into the docs. This PR has two topic pages. Do you have an opinion on one entry point vs two?

**Comment by grantgoodmanedX** [comment]
For a single entry point, we should use this URL:
https://support.edx.org/hc/en-us/sections/360001216693-Learner-records

File: en_us/help_center/programs/view-share-record.html [hunk 1]
```diff
@@ -0,0 +1,10 @@
+<p>After you have earned a certificate, you can view your certificates on the <strong>My Records</strong> page. The <strong>My Records</strong> page displays your course completion status for any courses that are part of a program, such as a MicroMasters. On the <strong>My Records</strong> page, you can choose to share your course record with an institution that accepts that course as a step towards admission or completion of a degree program.</p>
+<p>Your <strong>My Records</strong> page exists only if you have earned a certificate in a course that is part of a program.</p>
+<p>To share your course record, follow these steps.</p>
+<ol>
+<li>At the top of any page, select <strong>Dashboard</strong> from the dropdown menu icon next to your username, then select <strong>Profile</strong>. If you are already on your <strong>Dashboard</strong> page, select the <strong>Profile</strong> tab.</li>
+<li>On the <strong>Profile</strong> tab, select <strong>My Records</strong>.</li>
```
Comment (mikix) [inline en_us/help_center/programs/view-share-record.html]: "View My Records" not "My Records"

File: en_us/help_center/programs/view-share-record.html [hunk 2]
```diff
@@ -0,0 +1,10 @@
...
+<li>On the <strong>My Records</strong> page, find the course record that you want to share and select <strong>View record</strong>.</li>
```
Comment (mikix) [inline en_us/help_center/programs/view-share-record.html]: "find the program record" (not course record, we only currently support programs)
"View Program Record" (not "View record")

File: en_us/help_center/programs/view-share-record.html [hunk 3]
```diff
@@ -0,0 +1,10 @@
+<p>After you have earned a certificate, you can view your certificates on the <strong>My Records</strong> page. The <strong>My Records</strong> page displays your course completion status for any courses that are part of a program, such as a MicroMasters. On the <strong>My Records</strong> page, you can choose to share your course record with an institution that accepts that course as a step towards admission or completion of a degree program.</p>
```
Comment (mikix) [inline en_us/help_center/programs/view-share-record.html:1]: You use "My Records" to describe functionality available on program record detail pages in this document. If that's an intentional simplification, that's fine. But wanted to call it out.

File: en_us/help_center/programs/view-share-record.html [hunk 4]
```diff
@@ -0,0 +1,10 @@
...
+<p>Your <strong>My Records</strong> page exists only if you have earned a certificate in a course that is part of a program.</p>
+<p>To share your course record, follow these steps.</p>
+<ol>
+<li>At the top of any page, select <strong>Dashboard</strong> from the dropdown menu icon next to your username, then select <strong>Profile</strong>. If you are already on your <strong>Dashboard</strong> page, select the <strong>Profile</strong> tab.</li>
+<li>On the <strong>Profile</strong> tab, select <strong>My Records</strong>.</li>
+<li>On the <strong>My Records</strong> page, find the course record that you want to share and select <strong>View record</strong>.</li>
+<li>On the <strong>Record</strong> page for the course, select <strong>Request Credit</strong>.</li>
+</ol>
+<p>After you select <strong>Request Credit</strong>, the institution you shared your course record with can view the course record for that course. For detailed information about who can see your course record and what they can see, see <a href="https://support.edx.org/hc/en-us/articles/360004623354">When I share my course record, what information is shared?</a></p>
```
Comment (mikix) [inline en_us/help_center/programs/view-share-record.html]: "Send Learner Record" is the current UI text I believe, not "Request Credit" -- but Marco can confirm.

File: en_us/help_center/programs/what-info.html [hunk 5]
```diff
@@ -0,0 +1,13 @@
+<p>You can view and share your course record on the <strong>My Records</strong> page of the LMS. When you share your course record, the institution you share it with receives the following information about your performance in the course.</p>
```
Comment (mikix) [inline en_us/help_center/programs/what-info.html]: "program record" might be better than "course record"?

File: en_us/help_center/programs/what-info.html [hunk 6]
```diff
@@ -0,0 +1,13 @@
...
+<ul>
+<li>Your name.</li>
+<li>The course name.</li>
+<li>The name of the program that the course is a part of.</li>
+<li>The institution that offered the course.</li>
+<li>The number of times you started the course as a verified learner.</li>
+<li>The start date for the course.</li>
+<li>The end date for the course.</li>
```
Comment (mikix) [inline en_us/help_center/programs/what-info.html]: The current mockup doesn't show start/end dates. https://projects.invisionapp.com/share/ADK3TZQX3GP#/screens/300683943

File: en_us/help_center/programs/what-info.html [hunk 7]
```diff
@@ -0,0 +1,12 @@
+<p>You can view and share your program record on the <strong>My Records</strong> page of the LMS. When you share your program record, the institution you share it with receives the following information about your performance in each of the program's courses.</p>
+<ul>
+<li>Your name.</li>
+<li>The course name.</li>
+<li>The name of the program that the course is a part of.</li>
+<li>The institution that offered the course.</li>
+<li>The number of times you started the course as a verified learner.</li>
+<li>The date a certificate was awarded to you for the course.</li>
```
Comment (mikix) [inline en_us/help_center/programs/what-info.html:8]: We actually don't show this with current mockups. We only show when the program record as a whole was last updated.

File: en_us/help_center/programs/what-info.html [hunk 8]
```diff
@@ -0,0 +1,12 @@
+<p>You can view and share your program record on the <strong>My Records</strong> page of the LMS. When you share your program record, the institution you share it with receives the following information about your performance in each of the program's courses.</p>
+<ul>
+<li>Your name.</li>
```
Comment (mikix) [inline en_us/help_center/programs/what-info.html:3]: Plus email and username.

**Pull request merged**